src/Day.js

```diff
@@ -43,10 +44,17 @@ export default function Day({
   if (empty) {
     return <div role="gridcell" aria-disabled className={className} />;
```

Comment:
The empty cell may also have a modifier's `style`. Thus, better put this lines after creating the style objects and add the `style` prop to it.